index.html

```diff
@@ -322,7 +322,7 @@ <h3>What is a Verifiable Credential?</h3>
 refers to the characteristic of a <a>credential</a> or <a>presentation</a>
 as being able to be <a>verified</a> by a <a>verifier</a>,
 as defined in this document. Verifiability of a credential does not imply
-that the <a>claims</a> encoded in it are evaluatable for their truth content.
+that the truth content of <a>claims</a> encoded in can be evaluated.
```

Comment:
sorry, introduced my own typo.
```suggestion
that the truth content of <a>claims</a> encoded therein can be evaluated.
```